screens/tabbar/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { FC } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Home from '../home';
+import Accounts from '../accounts';
+import Giving from '../giving';
+import Payments from '../payments';
+import Cards from '../cards';
+import { Image } from 'react-native-elements/dist/image/Image';
+
+import { BlurView } from 'expo-blur';
+const Tab = createBottomTabNavigator();
+
+const TabBar: FC = () => {
+	return (
+		<Tab.Navigator
+			screenOptions={{
+				tabBarActiveTintColor: '#F08080',
+				tabBarBackground: () => <BlurView tint="light" intensity={100} style={StyleSheet.absoluteFill} />,
+				headerShown: false,
+			}}>
+			<Tab.Screen
```

Comment:
Please try to avoid any copy/paste. just create an array (e.g in a constants.ts) with necessary data and display it with .map method.